README.md

```diff
@@ -1 +1,13 @@
-# ci-files
\ No newline at end of file
+# scipy-ci-artifacts
+
+This repository is intended for use by the scipy/scipy repository continuous integration
+tests. We reserve the right to delete any files in this repository or the entire repository.
+
+## openblas binary for windows
+
+This binary is for a get-it-working-fast build needed when porting the Scipy build to Meson / mingw-w64.
+In particular, the openblas build needed to be built with the MinGW toolchain that SciPy is also
+built with. 
+
+The long-term plan is to work with everyone to get our OpenBLAS / Windows builds updated, and will likely
+result in something like the existing system.
```

Comment:
Yeah, this makes sense, presumably existing system will mean something like https://github.com/MacPython/openblas-libs and the scripts/staging areas we use related to that.

I don't think we need to painfully iterate on this now, the README will already be helpful for quick glance if someone sees a DL from this repo in CI and is curious.